packages/wxt/src/utils/content-script-ui/__tests__/index.test.ts

```diff
@@ -49,6 +49,7 @@ describe('Content Script UIs', () => {
         <p id="three"></p>
       </div>
     `;
+    // @ts-ignore
```

Comment:
Use specified ignore if it's possible, because JetBrains IDE(Maybe other also, but i have been using it) doesn't like it, and treating it as a warning

Reply:
Yeah, I'll probably convert this to a `@ts-expect-error`, was getting annoying errors while integrating bun's types into the project.